src/System.Net.Primitives/src/System/Net/IPEndPoint.cs

```diff
@@ -20,23 +20,23 @@ public class IPEndPoint : EndPoint
         ///       property.
         ///    </para>
         /// </devdoc>
-        public const int MinPort = 0x00000000;
+        public const int MinPort = UInt16.MinValue;
 
         /// <devdoc>
         ///    <para>
         ///       Specifies the maximum acceptable value for the <see cref='System.Net.IPEndPoint.Port'/>
         ///       property.
         ///    </para>
         /// </devdoc>
-        public const int MaxPort = 0x0000FFFF;
+        public const int MaxPort = UInt16.MaxValue;
 
         private IPAddress _address;
-        private int _port;
+        private UInt16 _port;
 
         internal const int AnyPort = MinPort;
 
-        internal static IPEndPoint Any = new IPEndPoint(IPAddress.Any, AnyPort);
-        internal static IPEndPoint IPv6Any = new IPEndPoint(IPAddress.IPv6Any, AnyPort);
+        internal static readonly IPEndPoint Any = new IPEndPoint(IPAddress.Any, AnyPort);
+        internal static readonly IPEndPoint IPv6Any = new IPEndPoint(IPAddress.IPv6Any, AnyPort);
```

Comment:
Are these actually used anywhere?  If not, they can instead be deleted.

Reply:
IDK, a simple compilation test should indicate if they are required, so I will delete it, and if it fails i will revert it.